src/main/java/core/basesyntax/impl/FruitsFromFileImpl.java

```diff
@@ -0,0 +1,27 @@
+package core.basesyntax.impl;
+
+import core.basesyntax.db.FruitShopDao;
+import core.basesyntax.service.FruitTransaction;
+import core.basesyntax.service.FruitsFromFile;
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class FruitsFromFileImpl implements FruitsFromFile {
+    private FruitShopDao fruitShopDao;
+
+    public FruitsFromFileImpl(FruitShopDao fruitShopDao) {
+        this.fruitShopDao = fruitShopDao;
+    }
+
+    @Override
```

Comment:
The method name `getFruits` is not descriptive enough to understand the purpose of this method. It should be renamed to something more specific to its functionality.